Doc/library/socket.rst

```diff
@@ -501,6 +501,19 @@ Constants
    .. availability:: Linux >= 2.2.
 
 
+.. data:: ETH_P_8021AD
+          ETH_P_8021Q
+          ETH_P_ALL
+          ETH_P_ARP
+          ETH_P_IP
+          ETH_P_IPV6
+
+   Many constants of these forms, documented in the Linux documentation, are
+   also defined in the socket module.
+
+   .. versionadded:: 3.12
```

Comment:
Should there be an `availability` directive as well?

Reply:
`ETH_P_IP` and `ETH_P_ARP` at least are available from before Linux 1.0